### PR TITLE
Issues/4

### PR DESCRIPTION
**Pull request by marrabld**
Fixes  [issue/4](https://github.com/AutomatedFishID/emtmlibpy/issues/4)

I placed the files in the wrong directory.

To test the fix on the branch
`pip install git+https://github.com/AutomatedFishID/emtmlibpy.git@issue/4`

If it works, then after it is merged to main

`pip install git+https://github.com/AutomatedFishID/emtmlibpy.git`

### CODE REVIEW

**Comment by nikoder** [comment]
Successfully installs both with `issues/4` and the explicit commit id as the ref.

`python -c "import emtmlibpy"` succeeds.

With the relevant files available at the expected paths, tests also pass:

```
$ LD_LIBRARY_PATH=$LD_LIBRARY_PATH:./ python -m unittest  test_emtmlibpy.py 
...................
----------------------------------------------------------------------
Ran 19 tests in 0.011s

OK
```
:+1: 

To my embarrassment, I haven't got Jupyter(Lab) installed yet. I'll whack that on and then have a look at the `ipynb`.

File: src/notebooks/gen_yolo_training_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "outputs": [],
+   "source": [
+    "SRC_DIR = os.path.abspath(os.getcwd())\n",
+    "TEST_FILES_PATH = os.path.join(SRC_DIR, '..', 'test_files')"
```
Comment (nikoder) [inline src/notebooks/gen_yolo_training_data.ipynb:41]: This is obviously fine, but I just thought I'd mention [`pathlib`](https://docs.python.org/3/library/pathlib.html#id5), which is a very nice standard library replacement for all of the old `os.path` helpers. E.g. you could use here:

```suggestion
    "import os, sys\n",
    "from pathlib import Path\n",
    "sys.path.append(\"..\")\n",
    "\n",
    "import emtmlibpy.emtmlibpy as emtm\n",
    "from emtmlibpy.emtmlibpy import EMTMResult\n",
    "\n",
    "import pandas as pd\n"
   ],
   "metadata": {
    "collapsed": false,
    "pycharm": {
     "name": "#%%\n"
    }
   }
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "outputs": [],
   "source": [
    "SRC_DIR = Path.cwd().resolve()\n",
    "TEST_FILES_PATH = SRC_DIR / '..' / 'test_files'"
```
With a small caveat that the above would also resolve symlinks, while `os.abspath()` would not. Presumably that would be of no concern, of course.

File: src/notebooks/gen_yolo_training_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "# this needs to be called first to generate the list\n",
+    "n_fgs = emtm.em_unique_fgs()\n",
+    "fgs = []\n",
+    "\n",
+    "for ii in range(n_fgs):\n",
+    "    fgs.append(emtm.em_get_unique_fgs(ii))\n",
```
Comment (nikoder) [inline src/notebooks/gen_yolo_training_data.ipynb:141]: Obviously a matter of taste, but I'd be tempted to use list or set comprehension here. E.g.:
```suggestion
    "fgs = {emtm.em_get_unique_fgs(ii) for ii in range(n_fgs)}\n",
```
(Assuming a set is appropriate for your use.)

Comment (marrabld) [reply src/notebooks/gen_yolo_training_data.ipynb]: It's like this right now so I can brearkpoint the list and inspect all the rows

**Comment by marrabld** [comment]
The notebook shouldn't be in this branch.  I have removed it and will commit to a new branch.

**Pull request merged**